app/views/exercises/show.html.slim

```diff
@@ -7,10 +7,20 @@ h1
   - if policy(@exercise).edit?
     = render('shared/edit_button', object: @exercise)
 
+
 = row(label: 'exercise.title', value: @exercise.title)
 = row(label: 'exercise.user', value: link_to_if(policy(@exercise.author).show?, @exercise.author, @exercise.author))
 = row(label: 'exercise.description', value: render_markdown(@exercise.description))
-= row(label: 'exercise.execution_environment', value: link_to_if(policy(@exercise.execution_environment).show?, @exercise.execution_environment, @exercise.execution_environment))
+= row(label: 'exercise.execution_environment') do
+  - if @exercise.execution_environment
```

Comment:
This block looks nearly identical to https://github.com/openHPI/codeocean/pull/132/files#diff-65b4469e16350935fe943bda56060ae2R32 -- it might be good to extract the template and `render` it here as a component